dependency-check-maven-config.xml

```diff
@@ -1,5 +1,94 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://www.owasp.org/index.php/OWASP_Dependency_Check_Suppression">
+
+    <!-- tika -->
+    <suppress>
+        <notes>
+            This is an issue with the jmatio library included in the tika-parser 1.13. The problem
+            doesn't exist in 1.14/1.15 so this is a false positive.
```

Comment:
❓ Would the problem go away if we upgraded to tika parser 1.14 or 1.15? If so, we may want to write up a ticket to upgrade and revisit this vulnerability.